util/Setup/Program.cs

```diff
@@ -195,7 +195,7 @@ private static void MigrateDatabase(int attempt = 1)
 
         migrator.MigrateMsSqlDatabaseWithRetries(log);
 
-        migrator.MigrateMsSqlDatabaseWithRetries(log, true, MigratorConstants.TransitionMigrationsFolderName);
+        migrator.MigrateMsSqlDatabaseWithRetries(log, true, false, MigratorConstants.TransitionMigrationsFolderName);
```

Comment:
❌ Unnecessary to provide this as it's optional.

Reply:
I would instead write `folderName: MigratorConstants.TransitionMigrationsFolderName`. It's still better to avoid passing the default. This is often why it's best to add new optional parameters _only_ to the end of method signatures.